velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -74,7 +74,8 @@ class HiveConnectorTestBase : public OperatorTestBase {
   makeHiveConnectorSplits(
       const std::string& filePath,
       uint32_t splitCount,
-      dwio::common::FileFormat format);
+      dwio::common::FileFormat format,
+      const dwio::common::ReaderOptions& readerOpts);
```

Comment:
Please revert changes in this file as they are no longer needed

Reply:
Done.  Waiting on local build and UT.